airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -26,12 +31,24 @@ public class JsonSchemaValidator {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(JsonSchemaValidator.class);
 
-  private final SchemaValidatorsConfig schemaValidatorsConfig;
   private final JsonSchemaFactory jsonSchemaFactory;
+  private final URI BASE_URI;
 
   public JsonSchemaValidator() {
-    this.schemaValidatorsConfig = new SchemaValidatorsConfig();
+    // This URI just needs to point at any path in the same directory as /app/WellKnownTypes.json
+    // It's required for the JsonSchema#validate method to resolve $ref correctly.
+    this("file:///app/nonexistent_file.json");
```

Comment:
What is this `BASE_URI` for? Could be good to have some explanation here, it's not obvious why we only want to expose this for tests given its current usage.

Reply:
added some comments - lmk if this is still unclear

Reply:
So if I understand correctly, we pass a value so that the validation can generate local references that we ignore anyway?

Reply:
the references aren't ignored - see the new unit test where the validator _does_ read the referenced file